service/frontend/admin_handler.go

```diff
@@ -1080,12 +1083,12 @@ func (adh *AdminHandler) ListClusterMembers(
 	heartbitRef := request.GetLastHeartbeatWithin()
 	var heartbit time.Duration
 	if heartbitRef != nil {
-		heartbit = *heartbitRef
+		heartbit = heartbitRef.AsDuration()
 	}
 	startedTimeRef := request.GetSessionStartedAfterTime()
 	var startedTime time.Time
 	if startedTimeRef != nil {
-		startedTime = *startedTimeRef
+		startedTime = startedTimeRef.AsTime()
```

Comment:
It's a time.Time because the persistence layer requires it and I tried my best to keep all the proto types out of the persistence API. Yichao agreed with that approach when I began

Reply:
I am ok with this. Let's use go types everywhere in code. I think other places needs to be cleaned up as too. Separate PR.